packages/api/src/models/commits/commits.rs

```diff
@@ -0,0 +1,27 @@
+use crate::Result;
+
+#[cfg(feature = "server")]
+use by_axum::aide;
+use by_macros::api_model;
+use by_types::QueryResponse;
+
+#[api_model(base = "/v1/commits", table = commits)]
+pub struct Commit {
+    #[api_model(primary_key, read_action = find_by_id)]
```

Comment:
```suggestion
    #[api_model(primary_key)]
```
you don't need to add find by id because we can utilize `query_builder`.
The `find_by_id` had been used before proposing `query_builder`.